core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -213,23 +216,22 @@ var ExplorationEditorPage = function() {
     }
   };
 
-  this.saveChanges = async function(commitMessage) {
-    var toastSuccessElement = element(by.css('.toast-success'));
-    await action.waitForAutosave();
+  this.saveChanges = async function(commitMessage, isPublished = false) {
```

Comment:
Hmm. How about splitting this function into two functions: saveChanges and publishChanges? From an API design perspective, it is not really intuitive that the "false" in saveChanges(false) refers to publication status.